newportxps/XPS_C8_drivers.py

```diff
@@ -1987,7 +1401,10 @@ def HardwareInternalListGet (self, socketId):
 
     # HardwareDriverAndStageGet :  Smart hardware
     def HardwareDriverAndStageGet (self, socketId, PlugNumber):
-        return self.Send(socketId, 'HardwareDriverAndStageGet(%s, char *, char *)' % str(PlugNumber))
```

Comment:
This would have parsed the outputs incorrectly before - would have returned one string instead of two.